### ISSUE TITLE
Add option to disable history

### ISSUE THREAD

**Issue by theotherp** [theotherp/nzbhydra#186]
Don't save a history of the searches if disabled.

**Comment by PuulPack** [comment]
Yes, please! I tried making nzbhydra.db, which I believe is the file where the history is written to, read only, but then Hydra would spit out an error when searching.
Hopefully this is easy to implement.

**Comment by theotherp** [comment]
Yeah, that won't work.

Please note that this will still leave traces of your activities. The log will be written and the database will contain each and every search result of the last 7 days or so. If you don't want to do that you need to stop using the program.

It would be a lot easier if you would just disable access to the stats.
